src/pages/Dashboard/Markets/SupplyMarket.tsx

```diff
@@ -127,7 +126,7 @@ const SupplyMarket: React.FC = () => {
       throw error;
     },
   });
-  const comptrollerContract = useComptroller();
```

Comment:
I literally just opened a PR updating this 😂

Reply:
I know, I've asked you multiple times to review the PRs lol